Makefile

```diff
@@ -70,6 +70,10 @@ lint-and-test-examples:
 	poetry run mypy --config-file mypy.ini docs/examples
 	poetry run flake8 --max-line-length=200 docs/examples
 	cd docs/tools && poetry run python prepare_examples_tests.py
+	# install duckdb extension for postgres_to_postgres example
```

Comment:
since postgres is now official extension IMO all those scripts can be removed
https://duckdb.org/docs/extensions/postgres

you just execute
```sql
INSTALL postgres;
```

and that we can do in the example script. this will be a huge relief because the code below binds the extension to particular duckdb version and will prevent many people from running the script (needs wget, extension works only on linux)